database-management/migrations/20180626110138_add_fulltext_index_to_articles.js

```diff
@@ -0,0 +1,12 @@
+exports.up = async knex => {
+  await knex.raw(
+    'alter table articles add fulltext `article-search-index` (title, markdown)',
```

Comment:
It's an SQL custom that makes it easier to read to write all the commands in all caps. so `ALTER TABLE articles ADD FULLTEXT ...` makes it slightly easier to read so no real reason not to, but otherwise let's merge it :)